nominations/templates/admin/application_pdf.html

```diff
@@ -0,0 +1,182 @@
+{% extends "easy_pdf/base.html" %}
+
+
+
+{% block content %}
+    {% load static %}
+    <link rel="stylesheet" href="{% static "dist/css/main.min.css" %}?v=122">
+
+    <style>
```

Comment:
Suggest to put this stuff in css files unless there is a need to have it here.

Reply:
There was some CSS weirdness with this PDF generator where in-code styles were working and linked files were not.